CSS/styles.css

```diff
@@ -0,0 +1,5 @@
+.happy{
+  background-color: balck;
```

Comment:
There's a small typo in the `background-color` attribute value. It should be `black`